test/ios_coverage_test.sh

```diff
@@ -173,6 +183,13 @@ function test_standalone_unit_test_coverage_json() {
       grep '"name":"SharedLogic.m:-\[SharedLogic doSomething\]"'
 }
 
+function test_standalone_unit_test_coverage_coverage_manifest() {
+  create_common_files
+  do_coverage ios --test_output=errors --ios_minimum_os=9.0 --experimental_use_llvm_covmap --action_env=LCOV_MERGER=/usr/bin/true --spawn_strategy=local --sandbox_debug //app:coverage_manifest_test || fail "Should build"
+  assert_contains "SharedLogic.m:-\[SharedLogic doSomething\]" "test-testlogs/app/coverage_manifest_test/coverage.dat"
+  assert_contains "SF:./app/StandaloneTest.m" "test-testlogs/app/coverage_manifest_test/coverage.dat"
```

Comment:
@keith I am really surprised when testing this locally, I can never make it work. Even when feeding the same paths and artifacts to `llvm-cov export` I cannot get paths to match up and engage path-equivalence

Reply:
I am not sure where I am going wrong in these tests since I can get this working [here](https://github.com/bazelbuild/rules_apple/pull/1824/files#r1114530777). Is there some hidden default flags or is this using some older version of rules apple support?